js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -801,6 +886,8 @@ const KeyboardNavigationController = core.ViewController.inherit({
                 const isInteractiveTarget = $target && $target.not($cell).is(INTERACTIVE_ELEMENTS_SELECTOR);
                 const isEditor = !!column && !column.command && $cell.hasClass(EDITOR_CELL_CLASS);
                 const isDisabled = !isEditor && (!args.isHighlighted || isInteractiveTarget);
+
+                this._isNeedFocus = true;
```

Comment:
Why?

Reply:
It is not necessary to do this. I removed it.